site/src/components/breadCrumb/index.js

```diff
@@ -41,22 +42,18 @@ const StyledLink = styled.div`
 `;
 
 export default function BreadCrumb({ data }) {
-  const chain = process.env.REACT_APP_PUBLIC_CHAIN;
-  if (!chain) {
-    throw new Error("CHAIN is not defined");
-  }
-  const chainName = chains.find((item) => item.value === chain)?.name;
+  const setting = useSelector(chainSettingSelector);
 
   return (
     <Wrapper>
-      <BreadCrumbWrapper chain={chain}>
+      <BreadCrumbWrapper>
         <Link to={`/`}>
-          <StyledLink chain={chain}>{chainName}</StyledLink>
+          <StyledLink>{setting.name}</StyledLink>
```

Comment:
this naming is a little bit confusing,  please alter it either `chainSetting` or `.chainName`